Makefile

```diff
@@ -12,9 +12,10 @@ prereq:
 	apt update -y && apt upgrade -y
 	apt -y install virtualbox vagrant
 
-build: custom_ca
-	docker build . -t local/hive:$(branch)
-	docker tag  local/hive:$(branch) local/hive:latest
+build:
+	docker build . -t fredrikhgrelland/hive:${LAST_COMMIT_HASH}
+	docker tag fredrikhgrelland/hive:${LAST_COMMIT_HASH} fredrikhgrelland/hive:test
+	docker tag fredrikhgrelland/hive:${LAST_COMMIT_HASH} fredrikhgrelland/hive:latest
```

Comment:
```suggestion
```
What's the point of tagging it as latest as well?

Reply:
I thought about the next release, but then I faced with #30 , and I lost my idea :) . I will remove it

Reply:
Sounds good!